{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/local.py

```diff
@@ -1,6 +1,6 @@
-"""
+'''
 Settings for local development.
-"""
+'''
```

Comment:
I'd probably just remove this comment now, it doesn't serve much of a purpose, same goes for the one at the top of `base.py`.